tests/Unit/Helpers/ParallelAlgorithms/Interpolation/Examples.hpp

```diff
@@ -91,7 +91,8 @@ struct ExampleComputeVarsToInterpolate
   template <typename SrcTagList, typename DestTagList, size_t Dim>
   static void apply(
       const gsl::not_null<Variables<DestTagList>*> /*target_vars*/,
-      const Variables<SrcTagList>& /*src_vars*/, const Mesh<Dim>& /*mesh*/) {
+      const Variables<SrcTagList>& /*src_vars*/,
+      const Mesh<Dim - 1>& /*mesh*/) {
```

Comment:
This shouldn't be Dim - 1 I think?

Reply:
Oops, yeah I was just testing out that the other signature was correct by making this one incorrect, but forgot to undo it.